content/docs/tasks/traffic-management/ingress/secure-ingress-sds/index.md

```diff
@@ -462,6 +462,10 @@ the credentials of the ingress gateway by deleting its secret and creating a new
 The server uses the CA certificate to verify
 its clients, and we must use the name `cacert` to hold the CA certificate.
 
+{{< tip >}}
```

Comment:
@dseapy, I just came across your PRs for this feature as I was doing my own investigation into CRL support. I think the feature is exactly what I need, but I can't really tell how to use it based on these docs. A more complete example would be very helpful.

Reply:
I'm happy to add more, I mainly didn't want to distract or complicate things for those reading this guide without needing a CRL.  So I tried to match how line 463 says "use name `cacert`" which matches the original line 470.

So you just need to add:
```
--from-file=crl=/some/path/to/your-crl.pem
```
so the full example would be:
```
kubectl create -n istio-system secret generic httpbin-credential  \
--from-file=key=httpbin.example.com/3_application/private/httpbin.example.com.key.pem \
--from-file=cert=httpbin.example.com/3_application/certs/httpbin.example.com.cert.pem \
--from-file=cacert=httpbin.example.com/2_intermediate/certs/ca-chain.cert.pem \
--from-file=crl=/some/path/to/your-crl.pem
```